src/app/api/services/settingsService.ts

```diff
@@ -0,0 +1,25 @@
+/***********************************************************
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License
+ **********************************************************/
+import { THEME_SELECTION } from '../../constants/browserStorage';
+import { Theme } from '../../shared/contexts/themeContext';
+import { getSettingsInterface } from '../shared/interfaceUtils';
+
+// high contrast either comes from native app or storage;
+export const getHighContrastSetting = async (): Promise<boolean> => {
+    const settings = getSettingsInterface();
```

Comment:
NIT: I am seeing that deviceService and localRepoService has been using const api to initialize the interfaces. It is quite convinient for us to locate all the calls using this new pattern during search. Wondering if you want to make this variable naming consistent here as well.